lua/apisix.lua

```diff
@@ -446,6 +446,29 @@ function _M.http_balancer_phase()
     load_balancer(api_ctx.matched_route, api_ctx)
 end
 
+local function cors_admin()
+    local local_conf = core.config.local_conf()
+    if local_conf.apisix or not local_conf.apisix.enable_admin_cors then 
```

Comment:
will be always `true`, you should add test case for this PR.

Reply:
I found this problem during my self-test, but forgot to submit the code

Reply:
_M.init_worker() in lua/apisix/admin/init.lua also has the same problem.

Reply:
> _M.init_worker() in lua/apisix/admin/init.lua also has the same problem.

welcome create another PR to fix it.

Reply:
ok